source/pom.xml

```diff
@@ -69,7 +69,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <encoding>${source.encoding}</encoding>
```

Comment:
I don't think so. `source.encoding` is used to set both `project.build.sourceEncoding` and `project.reporting.outputEncoding`. Strictly speaking `source.encoding` is an unfortunate name as it's also used to set the output encoding.